src/commands/cluster-migrations.json

```
@@ -0,0 +1,67 @@
+{
+    "MIGRATIONS": {
```

```suggestion
    "GET-SLOT-MIGRATIONS": {
```
Also based off of viktor's comment. This comment can be addressed last.

---

What did I say?

---

`HISTORY` (from [`LATENCY`](https://valkey.io/commands/latency-history/)) or `GET` (from [`SLOWLOG`](https://valkey.io/commands/slowlog-get/))?

---

Do we have an issue with CLUSTER SLOTMIGRATIONS? Like CLUSTER SLOTS, CLUSTER NODES, CLUSTER SHARDS, etc?

---

Okay we are going with GETSLOTMIGRATIONS